docs/libraries/nodejs/examples.md

```diff
@@ -1,101 +1,148 @@
 # Examples
 
-There are several examples to show the usage of the library.
-
-> All examples can be found in [/bindings/nodejs/examples](https://github.com/iotaledger/wallet.rs/bindings/nodejs/examples/)
-
-## Setup
-First, setup your environment as follows:
+There are several examples to show the usage of the library. All examples below can be also found in [/bindings/nodejs/examples](https://github.com/iotaledger/wallet.rs/tree/develop/bindings/nodejs/examples)
 
+All examples below expect your custom password in the `.env` file:
+```bash
+SH_PASSWORD="here is your super secure password"
 ```
-git clone https://github.com/iotaledger/wallet.rs
-cd bindings/node/examples
-npm install # or `yarn`
-cp .env.example .env
+> Please note: In is not recommended to store passwords on host's environment variables or in the source code in a production setup! Please make sure you follow our [backup and security](https://chrysalis.docs.iota.org/guides/backup_security.html) recommendations for production use!
+
+## Account manager and individual accounts
+First of all, let's initialize (open) a secure storage for individual accounts (backed up by Stronghold by default) using `AccountManager` instance:
+```javascript
+{{ #include ../../../bindings/nodejs/examples/1-create-account.js }}
 ```
+* Storage is initialized under the given path (`./alice-database`)
+* Password is set based on your password in `.env` file (`manager.setStrongholdPassword(process.env.SH_PASSWORD)`)
+* Only during the initialization new database: stronghold mnemonic (seed) is automatically generated and stored by default (`manager.storeMnemonic(SignerType.Stronghold)`)
+* Needless to say, the seed should be set only for the first time. In order to open already initialized database, just a password is enough
 
-Add your custom password to the `.env` file.
+The storage is encrypted at rest and so you need a strong password and location where to put your storage. Please note: it is highly recommended to store `stronghold` password encrypted on rest and separated from `stronghold` snapshots.
 
-## 1. Example: Create an Account
+> Please note: deal with the password with utmost care
 
-Run:
-```
-node 1-create-account.js
-```
+Technically speaking, the storage means a single file called `wallet.stronghold`. The generated seed (mnemonic) serves as a cryptographic key from which all accounts and related addresses are generated.
```

Comment:
we've updated the wallet code so it uses RocksDB as account storage and stronghold only as a seed storage and backup file, can you change it here? Sorry about the trouble :/

Reply:
No worries. I changed it.
So now it seems that `account_manager.backup` only backup the file with seed, right? So what should be the correct way of restoring a storage? I tried `account_manager.import_accounts` which I am afraid does not work anymore?

So it seems that restoring account means to recreated accounts from scratch based on the seed file that was backed up earlier?